### ISSUE TITLE
Implement "relayRequestId" scheme and release the package

### ISSUE THREAD

**Issue by Siegrift** [api3dao/ois#6]
Port https://github.com/api3dao/airnode/pull/1286 to https://github.com/api3dao/ois

**Comment by Siegrift** [comment]
Moved to done, will be tested with https://github.com/api3dao/airnode/issues/1274

**Comment by kolenic-martin** [comment]
Can we perhaps move this to Blocked for the time being @Siegrift? Otherwise this will be sitting here for who-knows-how-long.

**Comment by Siegrift** [comment]
Yeah, we can do that. I'd also want a confirmation on the [message in Slack](https://api3workspace.slack.com/archives/C02AYRX8D89/p1656421146177109).

Let's move it to blocked until then. Also, the **package needs to be released**.

**Comment by amarthadan** [comment]
Package published. Can it be moved to Testing and assigned to @kolenic-martin?

**Comment by Siegrift** [comment]
No, (see: https://github.com/api3dao/ois/issues/6#issuecomment-1168669581). But we can now move this to DONE.

**Comment by Siegrift** [comment]
I just realized that the bundled package.json version is wrong (it uses 1.0.0 instead of 1.1.0). This is because the release instructions build the package before the version command is in place. I've created a fix in https://github.com/api3dao/ois/pull/10 (and made the release process more bulletproof and automatic)

@aquarat Can you check and re-release? Also, make sure to push the branch afterwards (Current `main` branch has is still versioned `1.0.0`)
